epitome/functions.py

```diff
@@ -56,11 +58,11 @@ def download_and_unzip(url, dst):
     if not os.path.exists(dst):
         os.makedirs(dst)
 
+    input_dst = dst
```

Comment:
why do we set it to itself?

Reply:
`input_dst` is accessed later at the end of the function to delete the nested directory in line 97. The alternative would be to rename the `dst` file path on line 62 to a different name, LMK what you prefer